antispam_bee.php

```diff
@@ -396,7 +396,7 @@ private static function _init_internal_vars()
 				'dnsbl'		=> __( 'DNSBL Spam', 'antispam-bee' ),
```

Comment:
If you are already making changes to the names, the DNSBL was an old spam database that we don't use anymore. This now uses StopForumSpam database. Maybe we should even change the key `dnsbl` and the method names to keep it consistent, not only the string.